src/components/ApplyProductDesigner.tsx

```diff
@@ -0,0 +1,182 @@
+              translatable to working, codable products.
+            </Typography>
+          </Box>
+
+          <Box mb={1}>
+            <Typography variant="h6">What we’re looking for</Typography>
+            <Typography variant="body1" gutterBottom className={classes.text}>
+              A great designer is someone who places the user at the forefront
+              of the ideation and creation process. We want candidates
+              interested in problem solving and are skilled at communicating
+              ideas to team members. Although having web design experience is a
+              bonus, we value candidates that showcase a learning spirit,
+              dedication to the iterative process, and a receptive nature to
+              understand product needs and developer capabilities.
+            </Typography>
+          </Box>
+
+          <Box mb={3}>
+            <Typography variant="h6">
+              Our ideal canditiate is someone who...
+            </Typography>
+            <Typography variant="body1" className={classes.text}>
+              <ul className={classes.list}>
+                <li>
+                  Motivated to learn new technologies and grow as a product
+                  designer
+                </li>
+                <li>Passionate about helping the Boston community</li>
+                <li>
+                  A strong communicator who facilitates collaboration amongst
+                  the production team
+                </li>
+                <li>
+                  Interested in being a part of the entire product life cycle:
+                  software inception to completion
+                </li>
+                <li>
+                  Dedicated to the C4C mission and has excellent follow through
+                </li>
+              </ul>
+            </Typography>
+          </Box>
+        </Grid>
+
+        <Grid item sm={4} className={classes.application}>
+          <Stepper
+            className={classes.stepper}
+            activeStep={-1}
+            orientation="vertical"
+          >
+            <Step key={1} active>
+              <StepLabel>
+                Prepare your resume and read through our website
+              </StepLabel>
+            </Step>
+            <Step key={2} active>
+              <StepLabel>Send in your application</StepLabel>
+            </Step>
+            <Step key={3} active>
+              <StepLabel>Wait for us to contact you!</StepLabel>
+            </Step>
+          </Stepper>
+          <Box paddingLeft={'24px'} marginBottom={'2em'}>
+            <Button
+              variant="contained"
+              color="primary"
+              href="http://c4cneu.com"
+              target="_blank"
+            >
+              <Typography variant="body1" className={classes.applyButton}>
+                Apply to be a Product Designer
+              </Typography>
+            </Button>
+          </Box>
+        </Grid>
+        <QuoteBlock
```

Comment:
QuoteBlock or TextQuoteBlock? Not really sure if we picked one over the other but I was thinking of just switching to TextQuoteBlock for the MVP

Reply:
Yeah I agree